### PR TITLE
Add MDAnalysisTests package

### PR DESCRIPTION
**Pull request by kain88-de**
@orbeckst & @richardjgowers 

Are you ok with being co maintainers of this package?

### CODE REVIEW

**Comment by richardjgowers** [comment]
Yep!

**Comment by orbeckst** [comment]
Yes, I am. 

--
Oliver Beckstein
email: orbeckst@gmail.com

File: recipes/MDAnalysisTests/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - mock
+
+test:
+  imports:
+    - MDAnalysisTests
+    - MDAnalysisTests.datafiles
+
+about:
+  home: https://github.com/MDAnalysis/mdanalysis
+  license: GPLv2
```
Comment (synapticarbors) [inline recipes/MDAnalysisTests/meta.yaml]: I think we prefer `GPL-2.0`. See the following list for abbreviations:
https://opensource.org/licenses/alphabetical

I'm happy to merge after that change.

**Comment by kain88-de** [comment]
This has timed out due to another package in the master branch.

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes Can someone restart the CircleCI build on this PR? I would but CircleCI wants access to all of my private repos and I don't feel comfortable allowing that.

**Comment by kain88-de** [comment]
@synapticarbors I could restart the build myself

**Comment by synapticarbors** [comment]
@kain88-de I forgot to mention that you may have to pull the current master into this branch to clear the backlog of recipes that were hanging out in master. A bug fix was implemented recently and most of it has been cleared out.

**Comment by kain88-de** [comment]
@synapticarbors I could have thought of that myself.

**Comment by kain88-de** [comment]
The AppVeyor failure looks like a http timeout. But that one can't be related to this package since I deactivated windows builds.

**Pull request merged**